website/docs/d/is_instance.html.markdown

```diff
@@ -155,6 +155,36 @@ In addition to all argument reference list, you can access the following attribu
      - `enabled` - (Boolean) Indicates whether the metadata service endpoint will be available to the virtual server instance.
      - `protocol` - (String) The communication protocol to use for the metadata service endpoint.
      - `response_hop_limit` - (Integer) The hop limit (IP time to live) for IP response packets from the metadata service.
+
```

Comment:
Missing example for instance with network_attachments?

Reply:
added